packages/block-library/src/post-title/index.php

```diff
@@ -35,7 +30,7 @@ function render_block_core_post_title( $attributes, $content, $block ) {
 
 	if ( isset( $attributes['isLink'] ) && $attributes['isLink'] ) {
 		$rel   = ! empty( $attributes['rel'] ) ? 'rel="' . esc_attr( $attributes['rel'] ) . '"' : '';
-		$title = sprintf( '<a href="%1$s" target="%2$s" %3$s>%4$s</a>', get_the_permalink( $post_ID ), esc_attr( $attributes['linkTarget'] ), $rel, $title );
+		$title = sprintf( '<a href="%1$s" target="%2$s" %3$s>%4$s</a>', get_the_permalink(), esc_attr( $attributes['linkTarget'] ), $rel, $title );
```

Comment:
I wonder why this change wasn't already present before. Is it an oversight or is there an intentional difference in behavior for why this function _should_ still have the ID passed?